Application/EdFi.Ods.Features/ChangeQueries/ExceptionHandling/SnapshotGoneExceptionTranslator.cs

```diff
@@ -22,14 +23,14 @@ namespace EdFi.Ods.Features.ChangeQueries.ExceptionHandling
     /// </summary>
     public class SnapshotGoneExceptionTranslator : IExceptionTranslator
     {
-        private readonly ISnapshotContextProvider _snapshotContextProvider;
+        private readonly IContextProvider<SnapshotContext.SnapshotUsage> _snapshotContextProvider;
```

Comment:
Reduce this to `SnapshotUsage` by including the namespace with a `using` statement above.